src/app/drive/components/ShareDialog/ShareDialog.tsx

```diff
@@ -129,6 +130,7 @@ const ShareDialog = (props: ShareDialogProps): JSX.Element => {
   const isOpen = useAppSelector((state: RootState) => state.ui.isShareDialogOpen);
   const isToastNotificationOpen = useAppSelector((state: RootState) => state.ui.isToastNotificationOpen);
   const itemToShare = useAppSelector((state) => state.storage.itemToShare);
+  const { onCloseDialog } = props;
```

Comment:
Just an idea for this, cause you can abstract this kind of things easily by thinking about this, this way:

A dialog is just a wrapper.

Meaning you don't have a `ShareDialog`, you have let's say a `ShareConfigView`, which you can fit inside multiple components, one of the components that you might use this in, is a let's call it `Dialog`, so you can do this:

```
<Dialog onClose={props.onClose}>
    <ShareConfig />
</Dialog>
```

And by doing this you:

- Removed a lot of props from a lot of components if you use this pattern
- Now your dialogs are an abstract UI box where you can fit other components
- You can now reuse the `ShareConfig` in other parts of the app if needed

Reply:
An example: https://www.radix-ui.com/primitives/docs/components/alert-dialog